tensorflow/python/kernel_tests/reader_ops_test.py

```diff
@@ -374,23 +384,23 @@ def testOneEpoch(self):
           header_bytes=self._header_bytes,
           record_bytes=self._record_bytes,
           footer_bytes=self._footer_bytes,
+          hop_bytes=self._hop_bytes,
```

Comment:
Please add another test method, e.g. `testOneEpochWithHopBytes()` to test the new behavior.  We need to restore this unit test to its original coverage, namely `hop_bytes == 0` by default and use `_Record()`, `_num_records`, etc.

Furthermore, I'd highly suggest to simplify the record setup by explicitly spelling out the whole content for a simple case.  Your example in the pull request description is much clearer.